monitoring/osd-understanding-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,27 @@
+[id="understanding-the-monitoring-stack"]
+= Understanding the monitoring stack
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: understanding-the-monitoring-stack
+
+toc::[]
+
+User-workload monitoring lets you monitor your own applications using a different Prometheus instance on your own worker nodes, without interfering with Red Hat Site Reliability Engineer (SRE) platform metrics.
+
+[NOTE]
+Follow the instructions in this section carefully to configure a supported Prometheus instance. Custom Prometheus instances are not supported by {product-title}.
```

Comment:
This assembly does not provide details on how to configure Prometheus. You could move this note to the appropriate location.

Reply:
This note needs to stay with that first paragraph because we tell them they can use their own Prometheus instance for monitoring. So we want them to know they have to follow everything in this chapter. So I changed "section" to "chapter" to avoid the assembly issue that you rightly pointed out. (We also mention the custom Prometheus issue again later on - but they really want this called out.)